osquery/dispatcher/scheduler.cpp

```diff
@@ -116,16 +116,16 @@ inline void launchQuery(const std::string& name, const ScheduledQuery& query) {
     return;
   }
 
-  VLOG(1) << "Found results for query (" << name << ") for host: " << ident;
+  VLOG(1) << "Found results for query: " << name;
```

Comment:
I'd like to know when a query executes and when it fails. I don't care too much if the query has results.